spec/support/factory.rb

```diff
@@ -4,7 +4,8 @@
   factory :user do
     sequence(:username) { |n| "jarlax#{n}" }
     sequence(:email) { |n| "jarlax#{n}@launchacademy.com" }
-    sequence(:encrypted_password) { |n| "abcdef1234#{n}" }
+    sequence(:password) { |n| "abcdef1234#{n}" }
+    sequence(:password_confirmation) { |n| "abcdef1234#{n}" }
```

Comment:
Changing this here would create a merge conflict. Can we address this in a future PR?